### PR TITLE
chore: update TS deps

### PR DESCRIPTION
**Pull request by shahms**
Address the lodash vulnerability.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+    "@types/jasmine": "^3.5.11",
+    "@types/node": "^12.12.50"
   },
   "dependencies": {
-    "google-protobuf": "^3.12.1",
-    "source-map-support": "^0.5.16",
-    "typescript": "^3.8.3"
+    "google-protobuf": "^3.12.2",
+    "source-map-support": "^0.5.19",
+    "typescript": "^3.9.7"
```
Comment (evmar) [inline package.json:14]: This last line is changing TS language versions (they don't obey semver).  It's not clear to me whether the TS compilation bits of your build infra obeys this package.json, but I wouldn't expect it to.  So I think you should revert this line in particular.

Comment (shahms) [reply package.json:14]: I tried doing this, but the version in `package-lock.json` remains unchanged. Is that a problem?

Also, the BUILD rules should obey this file (from WORKSPACE):
```
npm_install(
    name = "npm",
    package_json = "//:package.json",
    package_lock_json = "//:package-lock.json",
)
```

Comment (evmar) [reply package.json:14]: The version pin here (`^3.9.7`) I think allows for any "semver-compatible" version.  I think the right way to express our current dep is
`"~3.8"`
to say we accept patchlevels on that verison (not sure, I just read the docs now).

I believe https://github.com/kythe/kythe/blob/ade706d5409a9a997f85d14a1251033577f1f5a1/kythe/typescript/BUILD#L3 controls which compiler is used, but I am not too familiar with these build rules.

Comment (shahms) [reply package.json:14]: Thanks!

**Comment by shahms** [comment]
This look good Evan?

**Pull request merged**